modules/xmpp/ChatRoom.js

```diff
@@ -266,8 +267,10 @@ export default class ChatRoom extends Listenable {
             if (this.password) {
                 pres.c('password').t(this.password).up();
             }
-            if (this.options.billingId) {
-                pres.c('billingid').t(this.options.billingId).up();
+
+            // send the machineId with the initial presence
+            if (this.moderator.targetUrl) {
+                pres.c('billingid').t(Settings.machineId).up();
```

Comment:
Why do we use stop using the `billingId` in favour of `machineId`? I don't understand this change. Could you please give me some context?